grammar.js

```diff
@@ -456,16 +458,6 @@ module.exports = grammar({
       $._let_binding,
     ),
 
-    _binding_pattern: $ => choice(
-      $.value_identifier,
-      $.tuple_pattern,
-      $.record_pattern,
-      $.array_pattern,
-      $.list_pattern,
-      $.module_unpack_pattern,
-      $.unit
-    ),
```

Comment:
This rule was introduced in #111 to accept `list` as a valid variable, but it wasn't really necessary, going back to `$._pattern`